lib/sanitize.js

```diff
@@ -73,5 +73,41 @@ sanitizer.config = {
     // Allow YouTube iframes
     if (frame.tag !== 'iframe') return false
     return !String(frame.attribs.src).match(/^(https?:)?\/\/(www\.)?youtube\.com/)
+  },
+  transformTags: {
+    'td': sanitizeCellStyle,
+    'th': sanitizeCellStyle
+  }
+}
+
+// Allow table cell alignment
+function sanitizeCellStyle (tagName, attribs) {
+  // if we don't add the 'style' to the allowedAttributes above, it will be
+  // stripped out by the time we get here, so we have to filter out
+  // everything but `text-align` in case something else tries to sneak in
+  function cell (alignment) {
+    var attributes = attribs
+    if (alignment) {
+      attributes.style = 'text-align:' + alignment
+    } else {
+      delete attributes.style
+    }
+    return {
+      tagName: tagName,
+      attribs: attributes
+    }
+  }
+
+  // note: we can do simple .indexOf checks here because markdown-it always
+  // generates `text-align` styles without whitespace
+  var style = attribs.style || ''
+  if (style.indexOf('text-align:left') >= 0) {
+    return cell('left')
+  } else if (style.indexOf('text-align:right') >= 0) {
+    return cell('right')
+  } else if (style.indexOf('text-align:center') >= 0) {
+    return cell('center')
+  } else {
+    return cell()
```

Comment:
i think we should do the regexp to allow the white space, in the end it will be more robust, and more closely matching the github behavior. lemme know if you want any help with that